tests/javascripts/sharing-button.test.js

```diff
@@ -0,0 +1,195 @@
+import helpers from './support/helpers.js'
+import sharingButton from '../../src/assets/javascripts/sharing-button.mjs'
+
+const URL = 'https://www.gov.uk/alerts/3-may-2021'
+const htmlFixture = `
+      <div class="share-url">
+        <p class="govuk-body">
+          ${URL}
+        </p>
+      </div>`
+
+afterAll(() => {
+  // clear up methods in the global space
+  document.queryCommandSupported = undefined
+})
+
+afterEach(() => {
+  document.body.innerHTML = ''
```

Comment:
Minor: we don't do this in the Admin app, do we need it in these tests?

Reply:
I think we do it in quite a few places in the admin app (see https://github.com/alphagov/notifications-admin/search?q=document.body.innerHTML) but I might be missing your point here about its value in these tests maybe. What do you think?

Reply:
Ah, fair enough - we're looking in different places for our precedents. More objectively, I'd assumed that, like the module state, JSDOM was being reset for each test module. However, [it appears I'm mistaken](https://stackoverflow.com/questions/42805128/does-jest-reset-the-jsdom-document-after-every-suite-or-test).

In practice, we're likely to do an assignment to `innerHTML` if we're testing something else that makes use of it. On the other hand, it's good practice to clean up. I don't mind if we keep it, as it's sort of optional.